setup.py

```diff
@@ -189,6 +208,7 @@
     "streaming": [],  # for backward compatibility
     "dev": TESTS_REQUIRE + QUALITY_REQUIRE,
     "tests": TESTS_REQUIRE,
+    "metrics_tests": METRICS_TESTS_REQUIRE,
```

Comment:
NIT: maybe using hyphen instead of underscore, as we already have "apache-beam".